app/controllers/shelters_controller.rb

```diff
@@ -78,11 +78,17 @@ def reviewedit
   end
 
   def reviewupdate
+    if params[:content] == "" || params[:rating] == "" || params[:title] == ""
+      flash[:notice] = "Please fill in Title, Rating, and Review sections"
+      redirect_to "/shelters/#{params[:id]}/#{params[:review_id]}/edit"
```

Comment:
Should `params[:id]` be called on an instance instead of calling params? Will research.